irium/chain.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "irium",
+  "status": "upcoming",
+  "network_type": "mainnet",
+  "pretty_name": "Irium",
+  "chain_id": "irium-1",
+  "bech32_prefix": "irium",
+  "daemon_name": "iriumd",
+  "node_home": "$HOME/.irium",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "urim",
+        "fixed_min_gas_price": 0.001,
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.003
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      { "denom": "urim" }
+    ]
+  },
+  "genesis": {
+    "genesis_url": "https://raw.githubusercontent.com/iriumlabs/irium/main/networks/mainnet/genesis.json",
+    "genesis_hash": "99df0ab5e17c0dd46a08ffd4a2e6bb78ca7d2da4dcabc6d0c1d3e273483cacdf"
+  },
+  "codebase": {
+    "git_repo": "https://github.com/iriumlabs/irium",
+    "cosmos_sdk_version": "v0.47.3",
```

Comment:
This property is no longer accepted. Please use the updated "cosmos" object property and be sure to include it in the versions.json file (you don't have one yet)